src/app/pages/OriginsLaunchpad/components/EmailSubscribtionForm/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface SubscriptionForm {
+  email: string;
+  name: string;
```

Comment:
We don't need `name` in this specific form.

Reply:
Marketing team (and api) expects to have both email and name/discord pseudonym